src/libraries/Common/src/Interop/OSX/System.Native/Interop.iOSVersion.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
let's call this file `Interop.iOSSupportVersion.cs` so it's clearer that this isn't really iOS specific.